### DISCUSSION TITLE
Howto use /etc/udev/rules.d ?

### DISCUSSION THREAD

**Discussion by MSchmitzGeWeTe** [dorssel/usbipd-win#311] [Q&A] [answered]
Hi...

I've a rule definition file in /etc/udev/rules.d but it seems not working if I attach the device. Has someone a tip how I could get it working in combination with usbipd-win? Thanks in advance!

Kind regards,
Michael

**Comment by Diegorro98** [comment, marked as answer]
Maybe this comment in an Issue that I opened can help you: https://github.com/dorssel/usbipd-win/issues/321#issuecomment-1079721555
In my case the problem is that the rules are not loaded correctly for some reason. So if going to use a device that requires the rule I have defined, I run `$ sudo service udev restart`, but I need to do every time that the distro is booted.

**Reply by MSchmitzGeWeTe** [reply]
Thank you, this works fine! I've a startupWSL.cmd, which will start all I needed to avoid this problem:

@echo off
wsl -l -v
wsl /bin/false
sudo service udev restart
usbipd wsl list
usbipd wsl attach --busid 4-1
wsl /usr/.../startup.sh
